### DISCUSSION TITLE
non latin letters search in react-instantsearch-dom

### DISCUSSION THREAD

**Discussion by naqshbandhamza** [algolia/instantsearch#5310] [Q&A]
can searchbox tag of react-instantsearch-dom also detect non latin characters?? if yes then how??

**Comment by Haroenv** [comment]
Yes, it can! You can use any input method, like in another HTML input. If you can reproduce it not working on https://codesandbox.io/s/github/algolia/create-instantsearch-app/tree/templates/react-instantsearch?file=/src/App.js, feel free to give more information

**Reply by naqshbandhamza** [reply]
![Screenshot (320)](https://user-images.githubusercontent.com/85412839/163973486-a13ad4bf-64a8-464c-9424-e894db994573.png)
![Screenshot (321)](https://user-images.githubusercontent.com/85412839/163973807-86a04e35-ef45-4602-a810-21185af0918d.png)
![Screenshot (322)](https://user-images.githubusercontent.com/85412839/163973933-1fe83843-a2a2-42d8-942d-81e517d4dcb7.png)
thanks for taking the time to reply please checkout these screenshots i am using the <searchbox/> tag and when the data is in english it search but when not for example arabic in our case the search bar doesnt work

**Reply by naqshbandhamza** [reply]
searchbox tag*

**Reply by Haroenv** [reply]
From the screenshot I don't understand what you mean with "don't work". Does it also not work in the sandbox I sent?

**Reply by naqshbandhamza** [reply]
By don't work I mean it doesn't filter out results

**Reply by naqshbandhamza** [reply]
For example in one screenshot the data is in English so when I write something in the search bar it filters out all the results which contain the text

**Reply by naqshbandhamza** [reply]
But that is not the case when it comes to non Latin characters I.e when I write Arabic text to filter out results it doesnot filter

**Reply by naqshbandhamza** [reply]
Do you understand the problem now???

**Reply by Haroenv** [reply]
Does the same happen in the dashboard? Are the arabic text in the same attribute as the English ones?

**Reply by naqshbandhamza** [reply]
Yes

**Reply by naqshbandhamza** [reply]
in Arabic the attribute ,value both are in Arabic

**Reply by naqshbandhamza** [reply]
And I do submit the sedahcable fields for example are data contains attributes as name city etc I submit these attributes as searchable fileds and do the same in Arabic too

**Reply by Haroenv** [reply]
This sounds like a problem generally, and not with React InstantSearch in that case, I recommend you contact support@algolia.com, thanks!
